frontend/src/app/util/data.cljs

```diff
@@ -168,3 +168,10 @@
           nm (str/trim (str/lower name))]
       (str/includes? nm st))))
 
+(defn tap
+  "Works like rx/tap but for normal collections"
+  ;; Signature for transducer use
+  ([f]
+   (map #(do (f %) %)))
+  ([f coll]
+   (map #(do (f %) %) coll)))
```

Comment:
In `app.common.data` there are tap and tap-r functions, I think would be good to unify them.

Reply:
You're right. I'll remove mine, it's redundant